### DISCUSSION TITLE
[chrome] chrome.runtime no longer unconditionally defined.

### DISCUSSION THREAD

**Discussion by AndrewSouthpaw** [DefinitelyTyped/DefinitelyTyped#70010] [Issues with a @types package]
The libdef defines `chrome.runtime` as always present, but according to [this post](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/tCWVZRq77cg/m/KB6-tvCdAgAJ) and my own experience, `chrome.runtime` is not defined when no connectable extensions are present.

[a comment by an automated account is not shown]

**Comment by fregante** [comment]
This package should only be installed by extension projects, not websites. 

If you find the package in your node_modules you should be able to block its loading by setting `compilerOptions.types` to `["node"]` for example. 

It might be interesting to set up an entry point for websites that expect connectable extensions, but you might have to open a PR with a solution yourself since nobody seems to be interested in this question. 

For example the solution might involve setting `compilerOptions.types` to `["node", "chrome/externally_connectable"]`, once that file is created.
